docs/api/geometries/TorusBufferGeometry.html

```diff
@@ -45,7 +45,7 @@ <h2>Constructor</h2>
 		<h3>[name]([page:Float radius], [page:Float tube], [page:Integer radialSegments], [page:Integer tubularSegments], [page:Float arc])</h3>
 		<div>
 		radius — Default is 1. <br />
```

Comment:
Can you also change this line to 
```
radius — radius of the torus. Default is 1.
```
see https://github.com/mrdoob/three.js/issues/13155#issuecomment-359588152

Reply:
https://github.com/mrdoob/three.js/pull/13163/commits/b5032dd15b1860640180f60cf9c2bf5438ac08f6